notes.md

```diff
@@ -257,24 +257,51 @@ so that the `Matroska Player` can quickly switch from one to the other.
 
 ## Default flag
 
-The "default track" flag is a hint for a `Matroska Player` and **SHOULD** always be changeable
-by the user. If the user wants to see or hear a track of a certain kind (audio, video, subtitles)
-and hasn't chosen a specific track, the `Matroska Player` **SHOULD** use the first track
-of that kind whose "default track" flag is set to "1". If no such track is found,
-then the first track of this kind **SHOULD** be chosen.
+The "default track" flag is a hint for a `Matroska Player` indicating that a given track
+**SHOULD** be eligible to be automatically selected as the default track for a given
```

Comment:
The concept of "automatic selection" is tricky. The automatic selection may/should not be the same for impaired people who would prefer a higher preference for such tracks.
IMO it's up to the player to select what fits best the user. At the Matroska level we should only provide the information of which tracks does what.
That's why the `FlagDefault` makes also more sense as a `FlagOriginal`. That's a more useful information for the player/user.

The user may end up wanting the Original+Audio Impaired+Commentary track by default.

Reply:
Ultimately I intend this entire section to be advisory; "this is a reasonable algorithm that you can use as-is, or extend with additional functionality, or implement some subset of". Currently players are very inconsistent about how to implement any of this. It certainly doesn't help that the flags have been poorly-specified, but even with clear descriptions of what the individual flags do, I think it's helpful to provide an explanation of how they all end up used together, and having some sort of authoritative guidelines (even if they're fairly loose) would likely help with consistency across players, and thus with adoption of the flags by file authors.

If we decide not to describe an example algorithm, I'd suggest at least providing a few example sets of tracks, explain which flags a file author should set on which tracks, and show which ones a player would be expected to select by default in some example configuration cases.

Reply:
Erm, not the "default flag" section, the autoselection-algorithm section. Though the "default" flag is ultimately advisory (to the player) as well.

Reply:
If we provide guidelines, even if informative, they will become de-facto the standard way. Or someone will have to fork a better guideline if it has some flaws.

In the end if you look at web streaming platforms (supposedly the most flexible technology) they don't follow the same rules nor provide the same set of choices regarding the language, accessibility, or even audio channels.

So I'd rather let the player choose its strategy given the possibilities it has and explain which fields are useful to select the proper track(s). Some examples could be useful as long as they are not too complex that some players will not even implement those examples properly.